lib/shoe.rb

```diff
@@ -0,0 +1,16 @@
+class Shoe
+  attr_accessor :color, :size, :material, :condition
+  attr_reader :brand
+
+  BRANDS = []
+
+  def initialize(brand)
+    @brand = brand
+    BRANDS << brand if !BRANDS.include?(brand)
```

Comment:
Why do `!BRANDS.include?(brand)` when you COULD do `BRANDS << brand unless BRANDS.include?(brand)` that's what `unless` exists!